easybuild/easyconfigs/r/R-synchronicity-1.1.9.1-foss-2015b-R.3.2.3.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Author: Adam Huffman <adam.huffman@crick.ac.uk>
+# The Francis Crick Institute
+
+easyblock = 'RPackage'
+
+name = 'synchronicity'
```

Comment:
OK, the easyconfig file is in the wrong place (should be `r/R/synchronicity...eb` rather than `r/R/synchronicity...eb`), so the tests are not picking up on it...

I'm fixing that in #2530, once that is merged, Jenkins should fail on this PR.

@verdurin: please hold off fixing this issue until #2530 is merged and we can retrigger the tests so Jenkins gives a red light on this one...